### PR TITLE
fixed the issue 31087

### PR DESCRIPTION
**Pull request by samiura**
**Description:**  Bumped docker v25.0.4+incompatible

**Link to tracking Issue:** 31087

**Testing:** Integration tests, unit test
Screenshot_20240308_120615

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

File: .chloggen/fix-issue-31087.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+change_type: 'bug_fix'
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: 'internal/docker'
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note:
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: ['31087']
```
Comment (dmitryax) [inline .chloggen/fix-issue-31087.yaml:13]: Looks like a wrong issue https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/31087

Comment (dmitryax) [reply .chloggen/fix-issue-31087.yaml:13]: Oh, no. It's the right one. nvm

Comment (dmitryax) [inline .chloggen/fix-issue-31087.yaml:13]: I think a number is expected. That's probably why CI is failing
```suggestion
issues: [31087]
```

Comment (samiura) [reply .chloggen/fix-issue-31087.yaml]: thanks

**Review by dmitryax — APPROVED** [review]

**Pull request merged**